SimCalorimetry/HGCalSimProducers/src/HGCDigitizerBase.cc

```diff
@@ -122,7 +142,7 @@ void HGCDigitizerBase<DFr>::runSimple(std::unique_ptr<HGCDigitizerBase::DColl>&
         toa[i] = cell.hit_info[1][i] / rawCharge;
 
       //final charge estimation
-      float noise = CLHEP::RandGaussQ::shoot(engine, 0.0, noiseWidth);
+      float noise = (float)cellNoiseArray[i] * noiseWidth;
```

Comment:
it might be worth putting a switch here to go back to the old behavior, in case someone wants to run a test in the future.